Back-end/index.js

```diff
@@ -93,6 +93,19 @@ app.get("/likes/:id", async (req, res) => {
   res.json(likes);
 });
 
+app.get("/commentcount/:id", async (req, res) => {
```

Comment:
nit: comment_count

Reply:
are you suggesting to change it to comment_count? Just want to make sure.